### PR TITLE
#161290893 Setup Unit testing

### PR DESCRIPTION
**Pull request by hope-uwa**
#### What does this PR do?
- Setup Unit Testing with Mocha and Chai

#### Description of Task to be completed?
- Install development dependencies for unit testing
- Install cross-enc for setting node environment
- Sample test for unit testing

#### How should this be manually tested?
- Clone this [repo](https://github.com/andela/noldor-ah-backend)
- On terminal, run `npm install` to install dependencies
- On terminal, run `npm test` to run test

#### Any background context you want to provide?
[Mocha](https://mochajs.org/) and [Chai](https://www.chaijs.com/) are installed as dev-dependencies

#### What are the relevant pivotal tracker stories?
#161290893
#### Screenshots 
image

#### Questions:
N/A

### CODE REVIEW

File: server/src/test/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+import sample from './sample'
```
[the remark this answers is not in the document]
Comment (desaawa) [reply server/src/test/index.js]: Can we fix these style violations?

[the remark this answers is not in the document]
Comment (desaawa) [reply server/src/test/index.js]: @uwaelpis

[the remark this answers is not in the document]
Comment (hope-uwa) [reply server/src/test/index.js]: @awadesmoline Yes. 
The review was done 20 hours ago by houndci, that was before @65black houndci configurations PR was merged to the develop branch. It should be fixed when merged.

**Review by ovieokeh — COMMENTED** [review]
The style violations are there because this PR was raised before the `.hound.yml` file was added to the project. This caused Hound to use the default configuration setting.

**Comment by desaawa** [comment]
@65black you have merge conflicts

File: server/src/test/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (desaawa) [reply server/src/test/index.js]: Please resolve this @65black.

**Comment by oluseyianani** [comment]
@uwaelpis looks good.

**Pull request merged**